clusterctl/repository/config.yaml

```diff
@@ -14,3 +14,5 @@ providers:
 cert-manager:
   url: "${CLUSTERCTL_REPOSITORY_PATH}/cert-manager/v1.9.1/cert-manager.yaml"
 overridesFolder: "${CLUSTERCTL_REPOSITORY_PATH}/overrides"
+CLUSTER_TOPOLOGY: TRUE
+EXP_RUNTIME_SDK: TRUE
```

Comment:
Looking at yuvaraj's PR: https://github.com/ykakarap/kubecon-na-22-capi-lab/pull/13/files

I wonder if caps-lock TRUE works

Reply:
Oh - I haven't looked at his yet. I'll close this in favor of that.

caps-lock TRUE does work though :slightly_smiling_face:

Reply:
Good to know :D